en/08_Changelogs/5.3.0.md

```diff
@@ -120,6 +136,9 @@ This is particularly helpful if you need to update columns in one table to match
 
 - Passing a non-array `$fields` argument to both [`FieldList::addFieldsToTab()`](api:SilverStripe\Forms\FieldList::addFieldsToTab()) and [`FieldList::removeFieldsFromTab()`](api:SilverStripe\Forms\FieldList::removeFieldsFromTab()) has been deprecated.
 - The [`BaseElement::getDescription()`](api:DNADesign\Elemental\Models\BaseElement::getDescription()) method has been deprecated. To update the description of elemental blocks, use the [`description`](api:DNADesign\Elemental\Models\BaseElement->description) configuration property and the localisation API.
+- The [`RememberLoginHash::renew()`](api:SilverStripe\Security\RememberLoginHash::renew()) method has been deprecated without replacement, since the associated behaviour will be removed in 6.0.
+  - The `onAfterRenewToken` extension point within this method will be replaced with a new extension point in 6.0.
```

Comment:
```suggestion
  - The `onAfterRenewToken` extension point within this method will likely be replaced with a new extension point in 6.0.
```
Since that hasn't been resolved yet - that way we can merge this without having to wait for the ongoing conversation to resolve.